### PR TITLE
Fixed indentation bug 

### PR DESCRIPTION
**Pull request by sukhbir-singh**

### CODE REVIEW

File: app/views/disk_wizard/done.html.slim [hunk 1]
```diff
@@ -62,7 +62,7 @@ css:
           tbody
             - count = 0
             - @operations.each do |operation|
-            - count += 1
+            -   count += 1
```
Comment (cpg) [inline app/views/disk_wizard/done.html.slim]: Too much space here?

**Comment by cpg** [comment]
What was the actual error?

**Comment by sukhbir-singh** [comment]
Bug: https://bugs.amahi.org/issues/2395#change-7817

The log file attached with this bug gives this comment: "disk_wizard/app/views/disk_wizard/select_fs.html.slim:235: syntax error, unexpected keyword_ensure, expecting keyword_end ^ensure". So, it must be because of the wrong indentation in the loop and if-else consider in slim file.

Log file link: https://paste.fedoraproject.org/paste/hNL4WMowqbnrsLD3lJS8PA

File: app/views/disk_wizard/done.html.slim (hunk 1 above)
Comment (sukhbir-singh) [reply app/views/disk_wizard/done.html.slim]: okay, I will fix this

**Comment by sukhbir-singh** [comment]
@cpg I have tested this PR. The bug (https://bugs.amahi.org/issues/2395) is now solved. 
Although, this disk-wizard plugin has some problem with layouts. I will fix and send another PR for that.

**Pull request merged**